zhaquirks/xiaomi/WXKG01LM_mija_button.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import homeassistant.components.zha.const as zha_const
+from zigpy.profiles import PROFILES, zha
+from zigpy.zcl.clusters.general import Basic, Groups, OnOff, PowerConfiguration, Identify
+from zhaquirks.xiaomi import BasicCluster, PowerConfigurationCluster,\
+    TemperatureMeasurementCluster, XiaomiCustomDevice
+from zhaquirks import CustomCluster
+
+BUTTON_DEVICE_TYPE = 0x0104
+BUTTON_DEVICE_TYPE_REPLACEMENT = 0x6FF1
+XIAOMI_CLUSTER_ID = 0xFFFF
+
+_LOGGER = logging.getLogger(__name__)
+
+PROFILES[zha.PROFILE_ID].CLUSTERS[BUTTON_DEVICE_TYPE_REPLACEMENT] = (
+    [
+        BasicCluster.cluster_id,
+        OnOff.cluster_id
+    ],
+    [
+        BasicCluster.cluster_id,
+        Groups.cluster_id
+    ]
+)
+
+zha_const.DEVICE_CLASS[zha.PROFILE_ID].update(
```

Comment:
This isn't needed when the profile and the device type in the replacement sig are updated per comments